connectors/sql-stored-connector/pom.xml

```diff
@@ -180,9 +163,11 @@
         </executions>
       </plugin>
 
-      <!-- generate components meta-data and validate component includes documentation etc
+      <!-- 
+      generate components meta-data and validate component includes documentation etc
       Disabled until the plugin supports adding parameters at the connector level. At the moment
-     they are added manually.
+      they are added manually.
```

Comment:
perhaps now we can remove this comment about the plugin being disabled?

Reply:
haha - good find :)